src/public/parts/shells/chat/public/src/templates/hub/profile_edit_modal.html

```
@@ -19,87 +19,87 @@ <h2 class="profile-edit-title" data-i18n="chat.profile.edit"></h2>
-						<div id="profile-edit-avatar-swatch" class="profile-edit-avatar-swatch"></div>
-						<div class="profile-edit-media-inputs">
-							<input type="url" id="profile-edit-avatar-url" class="input input-bordered input-sm" data-i18n="chat.hub.profileEdit.avatarUrl" />
-							<input type="file" id="profile-edit-avatar-upload" accept="image/*" class="file-input file-input-bordered file-input-sm" data-i18n="chat.hub.profileEdit.avatarUpload" />
+					<div class="profile-edit-field-grid">
+						<div class="form-control profile-edit-field">
+							<label class="label py-1" for="profile-edit-avatar-url"><span class="label-text text-xs opacity-70" data-i18n="chat.profile.avatarLabel"></span></label>
+							<div class="profile-edit-avatar-row">
+								<div id="profile-edit-avatar-swatch" class="profile-edit-avatar-swatch"></div>
+								<div class="profile-edit-media-inputs">
+									<input type="url" id="profile-edit-avatar-url" class="input input-bordered input-sm" data-i18n="chat.hub.profileEdit.avatarUrl" />
+									<input type="file" id="profile-edit-avatar-upload" accept="image/*" class="file-input file-input-bordered file-input-sm" data-i18n="chat.hub.profileEdit.avatarUpload" />
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)